react/queries/itemsWithSimulation.gql

```diff
@@ -0,0 +1,18 @@
+query itemsWithSimulation($items: [ItemInput]) {
+  itemsWithSimulation(items: $items) {
+    itemId
+    sellers {
+      commertialOffer {
+        Price
+        ListPrice
```

Comment:
It might be a good idea to add `spotPrice` here too

Reply:
This PR would solve this kind of issue:
https://github.com/vtex-apps/store-resources/pull/132

Reply:
spotPrice is not returned by the simulation API 🤔 
Maybe it is using another name? Here is a response example

```
{
  "items": [
    {
      "id": "11070",
      "requestIndex": 0,
      "quantity": 1,
      "seller": "1",
      "sellerChain": [
        "1"
      ],
      "tax": 0,
      "priceValidUntil": "2021-09-15T18:05:37.69419Z",
      "price": 11900,
      "listPrice": 14500,
      "rewardValue": 0,
      "sellingPrice": 11900,
      "offerings": [],
      "priceTags": [],
      "measurementUnit": "un",
      "unitMultiplier": 1.0000,
      "parentItemIndex": null,
      "parentAssemblyBinding": null,
      "availability": "available",
      "catalogProvider": "vrn:vtex.catalog-api-proxy:-:asimplertime:master:/proxy/authenticated/catalog/pvt/sku/stockkeepingunitbyid/11070"
    }
  ],
  "ratesAndBenefitsData": {
    "rateAndBenefitsIdentifiers": [],
    "teaser": []
  },
  "paymentData": {
    "installmentOptions": [
      {
        "paymentSystem": "1",
        "bin": null,
        "paymentName": "American Express",
        "paymentGroupName": "creditCardPaymentGroup",
        "value": 11900,
        "installments": [
          {
            "count": 1,
            "hasInterestRate": false,
            "interestRate": 0,
            "value": 11900,
            "total": 11900,
            "sellerMerchantInstallments": [
              {
                "id": "ASIMPLERTIME",
                "count": 1,
                "hasInterestRate": false,
                "interestRate": 0,
                "value": 11900,
                "total": 11900
              }
            ]
          }
        ]
      },
      {
        "paymentSystem": "2",
        "bin": null,
        "paymentName": "Visa",
        "paymentGroupName": "creditCardPaymentGroup",
        "value": 11900,
        "installments": [
          {
            "count": 1,
            "hasInterestRate": false,
            "interestRate": 0,
            "value": 11900,
            "total": 11900,
            "sellerMerchantInstallments": [
              {
                "id": "ASIMPLERTIME",
                "count": 1,
                "hasInterestRate": false,
                "interestRate": 0,
                "value": 11900,
                "total": 11900
              }
            ]
          }
        ]
      },
      {
        "paymentSystem": "3",
        "bin": null,
        "paymentName": "Diners",
        "paymentGroupName": "creditCardPaymentGroup",
        "value": 11900,
        "installments": [
          {
            "count": 1,
            "hasInterestRate": false,
            "interestRate": 0,
            "value": 11900,
            "total": 11900,
            "sellerMerchantInstallments": [
              {
                "id": "ASIMPLERTIME",
                "count": 1,
                "hasInterestRate": false,
                "interestRate": 0,
                "value": 11900,
                "total": 11900
              }
            ]
          }
        ]
      },
      {
        "paymentSystem": "4",
        "bin": null,
        "paymentName": "Mastercard",
        "paymentGroupName": "creditCardPaymentGroup",
        "value": 11900,
        "installments": [
          {
            "count": 1,
            "hasInterestRate": false,
            "interestRate": 0,
            "value": 11900,
            "total": 11900,
            "sellerMerchantInstallments": [
              {
                "id": "ASIMPLERTIME",
                "count": 1,
                "hasInterestRate": false,
                "interestRate": 0,
                "value": 11900,
                "total": 11900
              }
            ]
          }
        ]
      },
      {
        "paymentSystem": "5",
        "bin": null,
        "paymentName": "Discover",
        "paymentGroupName": "creditCardPaymentGroup",
        "value": 11900,
        "installments": [
          {
            "count": 1,
            "hasInterestRate": false,
            "interestRate": 0,
            "value": 11900,
            "total": 11900,
            "sellerMerchantInstallments": [
              {
                "id": "ASIMPLERTIME",
                "count": 1,
                "hasInterestRate": false,
                "interestRate": 0,
                "value": 11900,
                "total": 11900
              }
            ]
          }
        ]
      },
      {
        "paymentSystem": "12",
        "bin": null,
        "paymentName": "PayPal",
        "paymentGroupName": "payPalPaymentGroup",
        "value": 11900,
        "installments": [
          {
            "count": 1,
            "hasInterestRate": false,
            "interestRate": 0,
            "value": 11900,
            "total": 11900,
            "sellerMerchantInstallments": [
              {
                "id": "ASIMPLERTIME",
                "count": 1,
                "hasInterestRate": false,
                "interestRate": 0,
                "value": 11900,
                "total": 11900
              }
            ]
          }
        ]
      }
    ],
    "paymentSystems": [
      {
        "id": 2,
        "name": "Visa",
        "groupName": "creditCardPaymentGroup",
        "validator": null,
        "stringId": "2",
        "template": "creditCardPaymentGroup-template",
        "requiresDocument": false,
        "isCustom": false,
        "description": null,
        "requiresAuthentication": false,
        "dueDate": "2020-09-22T17:51:47.1848681Z",
        "availablePayments": null
      },
      {
        "id": 1,
        "name": "American Express",
        "groupName": "creditCardPaymentGroup",
        "validator": null,
        "stringId": "1",
        "template": "creditCardPaymentGroup-template",
        "requiresDocument": false,
        "isCustom": false,
        "description": null,
        "requiresAuthentication": false,
        "dueDate": "2020-09-22T17:51:47.1848681Z",
        "availablePayments": null
      },
      {
        "id": 4,
        "name": "Mastercard",
        "groupName": "creditCardPaymentGroup",
        "validator": null,
        "stringId": "4",
        "template": "creditCardPaymentGroup-template",
        "requiresDocument": false,
        "isCustom": false,
        "description": null,
        "requiresAuthentication": false,
        "dueDate": "2020-09-22T17:51:47.1848681Z",
        "availablePayments": null
      },
      {
        "id": 3,
        "name": "Diners",
        "groupName": "creditCardPaymentGroup",
        "validator": null,
        "stringId": "3",
        "template": "creditCardPaymentGroup-template",
        "requiresDocument": false,
        "isCustom": false,
        "description": null,
        "requiresAuthentication": false,
        "dueDate": "2020-09-22T17:51:47.1848681Z",
        "availablePayments": null
      },
      {
        "id": 5,
        "name": "Discover",
        "groupName": "creditCardPaymentGroup",
        "validator": null,
        "stringId": "5",
        "template": "creditCardPaymentGroup-template",
        "requiresDocument": false,
        "isCustom": false,
        "description": null,
        "requiresAuthentication": false,
        "dueDate": "2020-09-22T17:51:47.1848681Z",
        "availablePayments": null
      },
      {
        "id": 12,
        "name": "PayPal",
        "groupName": "payPalPaymentGroup",
        "validator": null,
        "stringId": "12",
        "template": "payPalPaymentGroup-template",
        "requiresDocument": false,
        "isCustom": false,
        "description": null,
        "requiresAuthentication": false,
        "dueDate": "2020-09-22T17:51:47.1848681Z",
        "availablePayments": null
      }
    ],
    "payments": [],
    "giftCards": [],
    "giftCardMessages": [],
    "availableAccounts": [],
    "availableTokens": []
  },
  "selectableGifts": [],
  "marketingData": null,
  "postalCode": null,
  "country": null,
  "logisticsInfo": [
    {
      "itemIndex": 0,
      "addressId": null,
      "selectedSla": null,
      "selectedDeliveryChannel": null,
      "quantity": 1,
      "shipsTo": [
        "USA"
      ],
      "slas": [],
      "deliveryChannels": [
        {
          "id": "delivery"
        }
      ]
    }
  ],
  "messages": [],
  "purchaseConditions": {
    "itemPurchaseConditions": [
      {
        "id": "11070",
        "seller": "1",
        "sellerChain": [
          "1"
        ],
        "slas": [],
        "price": 11900,
        "listPrice": 14500
      }
    ]
  },
  "pickupPoints": [],
  "subscriptionData": null,
  "totals": [
    {
      "id": "Items",
      "name": "Items Total",
      "value": 11900
    }
  ],
  "itemMetadata": null
}
```

Reply:
It's because the spotPrice is resolved only in the price resolver since it actually comes from the Installments! But in this case, it might not be necessary to add it (I am not 100% sure though, but if this becomes a problem, I think it is easy to fix!)